compiler/damlc/daml-doc/src/DA/Daml/Doc/Transform/Instances.hs

```diff
@@ -36,6 +38,15 @@ distributeInstanceDocs opts docs =
         [ (anchor, Set.singleton inst)
         | anchor <- Set.toList . getTypeAnchors $ id_type inst ]
 
+    getImplementsMap :: [ModuleDoc] -> ImplementsMap
+    getImplementsMap docs = Map.fromListWith Set.union
+        [ (anchor, Set.singleton iface)
+        | f <- md_functions =<< docs
+        , Just _ <- [T.stripPrefix "$implements_" . unFieldname . fct_name $ f]
```

Comment:
Can we change the prefix to `_implements_` instead of `$implements_`? That would make it easier to test e.g. in InterfaceDesugared.daml

Reply:
I went with `$implements_` since I worry about users adding spurious interfaces to a template type that doesn't really implement them, but maybe that's not actually a problem?

Reply:
I think it's fine because they would get an error ultimately, unless the template can actually implement the interface & they define all the other necessary bits. (And it still has to be in the same module as the template.)

Reply:
Done now :)

Reply:
Thanks!